src/components/stage/stage.jsx

```diff
@@ -112,68 +112,30 @@ const StageComponent = props => {
                             </div>
                         )}
                     </div>
-                    <canvas
-                        className={styles.draggingSprite}
-                        height={0}
-                        ref={dragRef}
-                        width={0}
-                    />
+                    {
+                        // Give the canvases two different keys to prevent React from recycling the same canvas
+                        // across two different context types (2D and WebGL)
+                        wobblyDragging ? <canvas
+                            className={classNames(styles.draggingSprite, styles.wobbly)}
+                            height={0}
+                            ref={wobblyDragRef}
+                            width={0}
+                            key="non-wobbly"
```

Comment:
Should these keys be switched?

Reply:
For ergonomics, yes. Did I introduce this last year and not notice?